src/lib/fs/path.c

```diff
@@ -293,3 +341,389 @@ make_path_absolute(char *fname)
   return absfname;
 #endif /* defined(_WIN32) */
 }
+
+/**** Helper functions of type glob_predicate_fn used for glob processing ****/
+
+/** Returns true if <b>file_type</b> represents an existing file (even if
+ * empty). Returns false otherwise. */
+static bool
+is_file(file_status_t file_type, const char *pattern_after)
```

Comment:
The is_file() function belongs in files.h, probably as an inline header function.

You should also use it in these functions, to replace existing logic:
* load_torrc_from_disk()
  * st == FN_FILE || st == FN_EMPTY
* find_torrc_filename()
  * nst == FN_FILE || st == FN_EMPTY
  * hmst == FN_FILE || hmst == FN_EMPTY

Reply:
Suggestion applied in ae426f9